packages/web-components/stencil.config.ts

```diff
@@ -1,14 +1,21 @@
 import { Config } from '@stencil/core';
 import { postcss } from '@stencil/postcss';
+import { sass } from '@stencil/sass';
 import url from 'postcss-url';
+import postcssSass from '@csstools/postcss-sass';
```

Comment:
This plugin [claims that its options are forwarded to the `sass` dependency](https://github.com/csstools/postcss-sass#options).